src/Database/Esqueleto/Internal/Internal.hs

```diff
@@ -3954,7 +4010,7 @@ renderExpr :: SqlBackend -> SqlExpr (Value Bool) -> T.Text
 renderExpr sqlBackend e = case e of
     ERaw _ mkBuilderValues ->
          let (builder, _) = mkBuilderValues Never (sqlBackend, initialIdentState)
-         in (builderToText builder)
+         in builderToText builder
```

Comment:
Thanks for the drive-by improvements!

Reply:
the most important one is the syntax fix on `sub_select` deprecation notice :smiley: (screenshot in PR head post)